src/VisualStudio/Core/Def/PdbSourceDocument/SourceLinkService.cs

```diff
@@ -35,7 +35,7 @@ internal class SourceLinkService : ISourceLinkService
             _logger = logger;
         }
 
-        public async Task<PdbFilePathResult?> GetPdbFilePathAsync(string dllPath, PEReader peReader, CancellationToken cancellationToken)
+        public async Task<PdbFilePathResult?> GetPdbFilePathAsync(string dllPath, bool useDefaultSymbolServers, PEReader peReader, CancellationToken cancellationToken)
```

Comment:
>useDefaultSymbolServers [](http://example.com/codeflow?start=87&length=23)

We usually pass bool flags as last parameters before cancellation token.

Reply:
> Woiuld it make sense to pass SymbolLocatorSearchFlags instead?

Can't access it from the caller, plus its too granular IMO, more like a debugger implementation detail. We could have our own enum obviously, but it would only have two values anyway.